README.md

```diff
@@ -32,3 +32,31 @@ found at the following links:
 
 + https://help.github.com/articles/creating-a-pull-request/ 
 + https://help.github.com/articles/autolinked-references-and-urls/
+
+
+## How to set up the API
+1. Before you can access the API, you need to install docker toolbox and docker-for-Mac (or whichever is appropriate for your machine)
+
+    https://www.docker.com/get-docker
+
+2. Once you have those installed, run the following command in your terminal:
+```sh 
+touch .env
+```
+This gives you access to the API's .env file. 
+
+3. Next you'll need to run the following:
+```sh
+  docker-compose build
+  docker-compose run --rm web bin/setup
+  docker-compose up web 
+```
+This last command starts the server. You'll need to keep this running in a separate tab.
+
```

Comment:
Since people working on the client will be doing steps 1-3, but nothing else, maybe put step 4. in a subheader `### Sending Requests to API via Postman`